docs/intro/Chapter0_Introduction.md

```diff
@@ -7,23 +7,23 @@ works. This introduction will guide you through some common use cases until you
 enough knowledge of the wrapper to be able to make more advanced and complex features.
 
 If you are new to Python then making a Discord bot (in any Python wrapper) is
-probably a bad idea. Creating a decent bot require intermediate Python knowledge,
-so to prevent frustration when learning, reading the documentation or understanding the 
+probably a bad idea. Creating a decent bot requires intermediate Python knowledge,
+so to prevent frustration when you are learning, reading the documentation or understanding the 
 offered help - it is a great idea to have solid Python knowledge.
 
-Hata wrapper itself should cover the whole open Discord API (OAuth2 too) and 
-it is designed to make bots fast and easy to create, it also comes with many
-additional functionality to make end-user life easier.
+The Hata wrapper itself should cover the entire Discord API (OAuth2 too), and 
+it is designed to make bots fast and easy to create, and it also comes with lots 
+of additional functionality to make user-end life easier.

```

Comment:
How about: `Hata itself covers the entire Discord API and is designed to make bots fast and easy to create, it also has lots of additional functionality to make the end-user's life easier.`